scripts/mssql_discover.py

```diff
@@ -0,0 +1,78 @@
+import sqlalchemy
+
+
+# NB. These are connection secrets, never ever commit these
+USER = ""
```

Comment:
Maybe add an option to read these from environment variables to make it less likely to commit them.